ui/pages/confirmations/confirmation/confirmation.js

```diff
@@ -507,103 +496,99 @@ export default function ConfirmationPage({
           </button>
         </div>
       )}
-      <div className="confirmation-page__content">
-        {templatedValues.networkDisplay ? (
-          <Box justifyContent="center" marginTop={2}>
-            <NetworkDisplay />
+      {isSnapCustomUIDialog ? (
+        <>
+          <Box width={BlockSize.Screen}>
+            <SnapAuthorshipHeader
+              snapId={pendingConfirmation?.origin}
+              onCancel={handleSnapDialogCancel}
```

Comment:
Why did we remove this https://github.com/MetaMask/metamask-extension/blame/develop/ui/pages/confirmations/confirmation/confirmation.js#L519-L525 ?

Reply:
Not removed, I just abstracted it